daemon/daemon_main.go

```diff
@@ -1408,9 +1408,9 @@ func (d *Daemon) instantiateAPI() *restapi.CiliumAPI {
 	api.PrefilterPatchPrefilterHandler = NewPatchPrefilterHandler(d)
 
 	// /ipam/{ip}/
-	api.IPAMPostIPAMHandler = NewPostIPAMHandler(d)
-	api.IPAMPostIPAMIPHandler = NewPostIPAMIPHandler(d)
-	api.IPAMDeleteIPAMIPHandler = NewDeleteIPAMIPHandler(d)
+	api.IpamPostIpamHandler = NewPostIPAMHandler(d)
```

Comment:
The fact that a code generator which deals with APIs thinks that changing case on *APIs* is OK is absurd to me... sigh.